website/docs/addon_resolve_admin.md

```diff
@@ -15,96 +15,51 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ## Resolve requirements
-Due to the way resolve handles python and python scripts there are a few steps required steps needed to be done on any machine that will be using AYON with resolve.
+Resolve is using its own python interpreter which cannot be connected to Ayon. This interpreter needs to have installed PySyde2 and OpenTimelineIO dependencies for Ayon to work correctly. Bellow you can find instructions on how to install them into an installed Python of your choice.
```

Comment:
This might be clarified in that it's not **Resolve's** interpreter. The issue with Resolve is actually that it does **not** have its own interpreter.

So the base of the message here would be:

- Resolve uses a separate Python interpreter (it does not come with Python itself)

That's also why you need to point it to a specific Python interpreter that is either installed locally or on a server. _If_ a user just install Py3 locally on their system there's actually no need to set the `RESOLVE_PYTHON3_HOME` because Resolve and Fusion will be able to detect system installed Python's if that var is not set.

Reply:
> If a user just install Py3 locally on their system there's actually no need to set the RESOLVE_PYTHON3_HOME because Resolve and Fusion will be able to detect system installed Python's if that var is not set.

This apply only for case that it is installed into specific path. It might be easyer to mention it this way just to streamline the information.